src/mrest.cpp

```diff
@@ -13,8 +13,9 @@ namespace vrv {
 // MRest
 //----------------------------------------------------------------------------
 
-MRest::MRest() : LayerElement("mrest-"), AttVisibility(), AttFermatapresent()
+MRest::MRest() : LayerElement("mrest-"), AttVisibility(), AttFermatapresent(), PositionInterface()
```

Comment:
This needs to be in the same order as in the header (some compilers will complain about it)